src/core/a-assets.js

```diff
@@ -83,9 +83,11 @@ registerElement('a-asset-item', {
     attachedCallback: {
       value: function () {
         var self = this;
-        var src = this.getAttribute('src');
+        var src;
 
+        src = this.getAttribute('src');
```

Comment:
So [in this function for instance](https://github.com/aframevr/aframe/blob/master/src/core/scene/a-scene.js#L264) do I have to do?
````javascript
function () {
  var camera;
  var canvas;
  var embedded;
  var size;
  camera = this.camera;
  canvas = this.canvas;
  embedded =  this.getAttribute('embedded') && !this.is('vr-mode');
}
`````
It feels like a lot of clutter.